packages/arb-token-bridge-ui/src/util/WithdrawalUtils.ts

```diff
@@ -60,7 +60,10 @@ export async function withdrawInitTxEstimateGas({
       estimatedChildChainGas
     }
   } catch (error) {
-    Sentry.captureException(error)
+    const code = (error as { code: string }).code
+    if (code !== 'INSUFFICIENT_FUNDS' && code === 'UNPREDICTABLE_GAS_LIMIT') {
```

Comment:
Are we sure we don't want to catch `UNPREDICTABLE_GAS_LIMIT` ? Imho We might want to know what's causing that error.

Reply:
We might also want to use `isUserRejectedError` here as well.

Reply:
Thinking about it, we might want to remove `isUserRejectedError` everywhere, and instead filter it in `Sentry.init`. I don't think we ever want to send an user rejected error?

Reply:
Updated. https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1682/commits/ce0568c7fab707dba20f798500e24b88ccdd67dd

It's now filtered in a centralized place

Reply:
I agree with Dragisa - we should remove this check `code !== 'INSUFFICIENT_FUNDS' && code === 'UNPREDICTABLE_GAS_LIMIT'` completely.
Is it bloating Sentry too much?

Reply:
It's ~90% of the errors